.helm/ecamp3/templates/browserless_deployment.yaml

```diff
@@ -0,0 +1,53 @@
+{{- if .Values.browserless.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "browserless.name" . }}
+  labels:
+    {{- include "app.labels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "app.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "app.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "app.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}-browserless
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.browserless.image.repository }}:{{ .Values.browserless.image.tag | default .Values.imageTag }}"
+          imagePullPolicy: {{ .Values.browserless.image.pullPolicy }}
+          ports:
+            - name: browserless-ws
+              containerPort: 3000
+              protocol: TCP
+          readinessProbe:
+            httpGet:
+              scheme: HTTP
+              path: /pressure
```

Comment:
Where is this endpoint documented? I only found the GraphQL pressure endpoint. Is this endpoint suited for a health check (it sure sounds like it, but do we have any official statement to that end)? Will this endpoint remain available in the image, or is it an undocumented feature?

Reply:
It was the best I could find to have some sort of indication that the image is up and running.

Issue https://github.com/browserless/chrome/issues/1420 specifies a better method. I believe we should follow progress of this issue and implement, once a better health endpoint is available.